### PR TITLE
De boilerplate zonder docs

### PR DESCRIPTION
**Pull request by wilbo**

### CODE REVIEW

**Comment by jjkramok** [comment]
Zooming by scrolling on android does not work. It pitches the view around the cube matrix instead.

File: core/src/com/marama/tests/entities/SelectableInstanceTest.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.marama.tests.entities;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class SelectableInstanceTest {
+
+    @Test
+    public void shouldUpdateMaterialWhenSelectableInstanceIsSelected() {
+        // TODO: make
```
Comment (jjkramok) [inline core/src/com/marama/tests/entities/SelectableInstanceTest.java]: TODO should either be fixed before PR or made into a new card, see DoD.

**Comment by jjkramok** [comment]
Approved

**Review by LuneCoding — APPROVED** [review]
Approved

**Pull request merged**